node/tracing-logging-and-metrics-for-sdk-and-core.md

```diff
@@ -0,0 +1,96 @@
+## Tracing, Logging, and Metrics in the NodeJS SDK and Core
+
+### What's already implemented in the Alpha release?
+
+Core [uses opentelemetry](https://github.com/temporalio/sdk-core#debugging) for generating logs and traces, it does not emit any metrics.
+
+The Node SDK uses `WorkerOptions.logger` directly for logging and the opentelemetry API for tracing with no output by default, it does not emit any metrics either.
+
+There's an example of how to export the traces to jaeger in [`bench.ts`](https://github.com/temporalio/sdk-node/blob/a25c0feca791a195eb5e50cf8c0fd9e0b5a1db83/packages/test/src/bench.ts#L105).
+
+See the [documentation](https://docs.temporal.io/docs/node/logging/) on how to customize the Worker logger.
+
+There's currently not a way to log from Workflow internals (inside the isolate).
+
+### Requirements
+
+- Link traces between Node and Core for a more complete picture
+- Core traces should be propagated through NodeJS
+- Core metrics should be propagated through NodeJS
```

Comment:
Do they actually need to be? I want to say as long as we use the same tags for metrics they'll get aggregated by the thing that receives them just fine.

Reply:
They have to be so users can export them using their current metrics solution (via lang)